### PR TITLE
Big red laststand

### PR DESCRIPTION
**Pull request by PalJohnson**
## About The Pull Request

- This update is seeking to shake up the survivor gameplay by providing unique scenarios for survivors. This update adds several survivor holdouts (neutral & hostile) for survivors. In addition, I added a minor bugfix for synth survivor,

This update got delayed due to gitkraken nonsense, the bot closing my mr, and etc. Sorry for delay

## Why It's Good For The Game

- One of the most underrated aspects of survivors is the ability to be able to do a last stand. The current meta shift revolves around LZ holds or roaming if that's not possible. This update should encourage survivors not to leave their place of work and instead fight and in most cases die for their company if they feel inclined to as opposed to roaming or cheesing out LZ holds.

## Non-Goals

- Giving survivors additional gear. Given that their gear is random and spread across the maps survivors will now have a focal point to restock on valued items in a localized area and runoff. Although survivors are expected to stay and fight to the last colonist, nothing stops them from looting the place. In the end of the day its player agency and I don't think it would be good to force them to do something they don't want to do like die in the bar, library or marshalls. Given that most the CLF insert have 25% spawn rate, as well as the dnd, insert this shouldn't be a problem. Given the frequency of the bar holdout, I have purposely given them fewer resources.
- 
- Xenomorphs have a harder time killing survivors. Although killing or capturing survivors is not a xenomorph goal as they can ignore them entirely. Having a holdout at places like bar or even libraries for that matter could prove to be a nuisance due to premade defenses and gear being available. As stated previously nothing guarantees the survivors staying there in the first place so they could be unmanned and easily melted. Don't assault a survivor holdout alone.

## Changelog

_**- Maps**_

This update as mentioned adds 3 new survivor holdouts for survivors that will randomly spawn throughout the map. Currently, there are only 3 being added but more will be added in the future. The colony didn't get raptured it was a bloody mess with people crying and fighting a hell hole. The MR has been balanced using the CLF crashed ship insert on LV-624 as an example as to how much medical, guns and other items will be spawned in for survivors. These are prepared positions so they should have ok gear and protection if we actually want to encourage people to hold it.

_**CLF Marshalls Take Over**_ **(25% Chance)**
image

- As stated previously the CLF have invaded the colony and have taken over the last bastion of the authority of the colony that is the colonial marshall's offices and armory. After the successful takeover, the CLF decided to make the ruins of the office into their own area of operation for future CLF activities in the colony. much like the crashed ship insert, they have been equipped with identical gear to their LV-624 counterparts with minor exceptions. The exception is materials to repair and potentially expand their area of influence. The barricades for the most part are worn out and need some repairs if they expect to live long enough.

_**Dungeons and Dragons Last Stand**_ **(25% Chance)**
image
- This nightmare insert was inspired by the event held by jarek where survivors attempted to have a sleepover in the LV nexus dormitories only to be slaughtered. This insert mirrors that in that colonists are hosting a dnd session in the local library. Given that the civilian survivor is rather lackluster I have gone ahead and supplied them with cosplay material so that they can arm themselves and carry out their dnd session in style. Much of it is fluff in many regards with the exception of the illegally smuggled merc gear. The balance of the insert is similar in the crashed ship insert but the barricades are repaired and the shutters closed.

_**The Bar Last Stand**_ **_(40% Chance)_**
image
- This insert reflects the valiant efforts of the colonist to have a last stand within the colony bar. The bar (according to the colonist handbook) was designated as an emergency rally point for all personnel and staff in the event of an unforeseen event. They come equipped with standard wooden barricades with building material should they want to replace them. In addition, they have ammo, medical supplies, and liquid courage to help them in their endeavor.

_**Minor Updates**_

- Synth survivor now has colonist radio. Previously in the synth survivor update, I accidentally gave them the wrong radio. This prevented them from communicating with survivors as they had a unique radio. The auto lathes produce marine radios so they couldn't find an adequate communication device.

- Colonist survivor now has the colonist jumpsuit. They had PJs for some reason. Odd

🆑 TheFlyingFlail
add: Added three new survivor holdout nightmare inserts for big red
add: Added clf survivor to big red with their own unique story to go with it
expansion: Expanded existing mechanics or gameplay
fix: fixed synth survivors not having colonists' radio
fix: fixed civilian survivor not having actual colonist clothing
/:cl:

### CODE REVIEW

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 1]
```diff
@@ -58,3 +58,17 @@
...
+
+/obj/effect/landmark/survivor_spawner/dnd
+	equipment = /datum/equipment_preset/survivor/civilian
+	intro_text = list("<h2>You are a civilian participating in a dungeons and dragons game!</h2>",\
+	"<span class='notice'>You are NOT aware of the xenomorph threat due to the rather engaging dnd game going on.</span>",\
+	"<span class='danger'>Your primary objective is to have a fun time and survive.</span>")
+	story_text = "You are a goup of colonist who regularly meet up to play dnd in the library every friday night. You alongside your fellow colonist brought various costumes, food, drinks, and props (very real guns) to help get you all into character. You aren't very exceptional in your skills but you do have the one thing that bests even the mightiest demons. That special power imagination!"
```
Comment (BadAtThisGame302) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:74]: group of colonists*

**Review by stanalbatross — COMMENTED** [review]
image
still need to do these changes from the last pr

**Review by PalJohnson — COMMENTED** [review]
reviewed changes

**Comment by stanalbatross** [comment]
fix files

**Comment by PalJohnson** [comment]
https://github.com/cmss13-devs/cmss13/pull/752 had to make new MR because of rebasing issue. Sorry

**Pull request closed without merging**